backend/pong/oauth2/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import oauth2_authorize, oauth2_callback, oauth2_fetch_token
+
+urlpatterns = [
+    # 'api/token/'
+    path("authorize/", oauth2_authorize, name="token_authorize"),
+    path("callback/", oauth2_callback, name="token_callback"),
+    path("token/", oauth2_fetch_token, name="token"),
+]
```

Comment:
[imo]
jwt の方の token に引っ張られてるかもです…！
コメントの URL と、name 部分は `oauth2` を接頭辞に使うと良さそうです
```suggestion
urlpatterns = [
    # 'api/oauth2/'
    path("authorize/", oauth2_authorize, name="oauth2_authorize"),
    path("callback/", oauth2_callback, name="oauth2_callback"),
    path("token/", oauth2_fetch_token, name="oauth2_fetch_token"),
]
```